spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -1031,9 +1031,6 @@ class CometExecSuite extends CometTestBase {
           |GROUP BY key
               """.stripMargin)
 
-      // The above query uses COUNT(DISTINCT) which Comet doesn't support yet, so the plan will
-      // have a mix of `HashAggregate` and `CometHashAggregate`. In the following we check all
-      // operators starting from `CometHashAggregate` are native.
       checkSparkAnswer(df)
```

Comment:
Can this change to check for Comet operators now?

```suggestion
      checkSparkAnswerAndOperator(df)
```